src/main/kotlin/inc/evil/d17map/dtos/ExtraRoomResponse.kt

```diff
@@ -9,5 +8,6 @@ class ExtraRoomResponse (
     val modelKey: String,
     val description: String,
     val type: String,
-    val floor: Floor
+    val floorName: String,
```

Comment:
W sumie czy floorName to jest u nas numer piętra? Jeśli tak to jest okej, a jak nie, to czym jest xD